packages/babel-core/test/config-chain.js

```diff
@@ -6,6 +6,27 @@ import util from "util";
 import escapeRegExp from "lodash/escapeRegExp";
 import * as babel from "../lib";
 
+// "minNodeVersion": "10.0.0" <-- For Ctrl+F when dropping node 10
+const supportsESM = parseInt(process.versions.node) >= 12;
+
+const isMJS = file => file.slice(-4) === ".mjs";
+
+const skipUnsupportedESM = (esm, name) => {
+  if (esm && !supportsESM) {
+    console.warn(
+      `Skipping "${name}" because native ECMAScript modules are not supported.`,
+    );
+    return true;
+  }
+  if (esm && process.platform === "win32") {
```

Comment:
May worthy link to the node upstream issue so we can eventually get rid of it in the future.

Reply:
https://github.com/babel/babel/pull/12288/files#diff-a182b20eac0484301d7882456ab81dc9b3235d95ebe2bcd2de1a8eeeeda9efc8R84